src/Fantomas.Tests/PatternMatchingTests.fs

```diff
@@ -587,8 +587,8 @@ match x with
         """
 match x with
 | Some y ->
-    let z = 1
```

Comment:
I'm not convinced by this. I think the feature should only affect the last block, and it shouldn't affect the *width* of an indentation at all - only the "number of tabs". This is the first block in the match, and not the last block, so I think it shouldn't change.

Reply:
This test is actually unrelated to the feature but also is a side effect of solving https://github.com/fsprojects/fantomas/issues/1501.